<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Widget</class>
 <widget class="QWidget" name="Widget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>716</width>
    <height>563</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>小马视频播放器</string>
  </property>
  <property name="windowIcon">
   <iconset resource="images.qrc">
    <normaloff>:/new/prefix1/images/u=1648998499,165919943&amp;fm=26&amp;gp=0.jpg</normaloff>:/new/prefix1/images/u=1648998499,165919943&amp;fm=26&amp;gp=0.jpg</iconset>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0" colspan="3">
    <widget class="QVideoWidget" name="widget" native="true"/>
   </item>
   <item row="1" column="0">
    <widget class="QToolButton" name="toolButton">
     <property name="text">
      <string>...</string>
     </property>
     <property name="icon">
      <iconset resource="images.qrc">
       <normaloff>:/new/prefix1/images/u=1129145644,412791288&amp;fm=26&amp;gp=0.jpg</normaloff>:/new/prefix1/images/u=1129145644,412791288&amp;fm=26&amp;gp=0.jpg</iconset>
     </property>
     <property name="iconSize">
      <size>
       <width>50</width>
       <height>50</height>
      </size>
     </property>
    </widget>
   </item>
   <item row="1" column="1">
    <widget class="QSlider" name="horizontalSlider">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </item>
   <item row="1" column="2">
    <widget class="QToolButton" name="toolButton_2">
     <property name="text">
      <string>...</string>
     </property>
     <property name="icon">
      <iconset resource="images.qrc">
       <normaloff>:/new/prefix1/images/5b33478ba1b05_610.jpg</normaloff>:/new/prefix1/images/5b33478ba1b05_610.jpg</iconset>
     </property>
     <property name="iconSize">
      <size>
       <width>50</width>
       <height>50</height>
      </size>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>QVideoWidget</class>
   <extends>QWidget</extends>
   <header location="global">qvideowidget.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="images.qrc"/>
 </resources>
 <connections/>
</ui>
